### ISSUE TITLE
my_softmax

### ISSUE THREAD

**Issue by cankucuksozen** [ethanfetaya/NRI#25]
Why does the my_softmax function seems to be normalizing alongside the batch dimension instead of classes dimension?

**Comment by wang422003** [comment]
Hi! Have you found any possible explanation for the my_softmax? I am quite confused about this one. Maybe I missed some details in the paper.

**Comment by ethanfetaya** [comment]
I think that was done because at that time the pytorch softmax did not have
a dim input variable as it does now.

**Comment by wang422003** [comment]
Many thanks!
